general_website/static/general_website/trees/balance.json

```diff
@@ -0,0 +1,639 @@
+[
+    {   "name": "Eclipse",
+        "description": "Casting 2 Starfires empowers Wrath for 15 seconds. Casting 2 Wraths empowers Starfire for 15 seconds. These Exlipses occur in alternation.\nEclipse (Solar): Wrath cast time reduced 8% and damage increased 20%.\nEclipse (Lunar): Starfire cast time reduced 8% and critical strike chance increased 20%.",
+        "max_rank": 1,
+        "coordinates": [
+            5,
+            1
+        ],
+        "child_coordinates": [
+            [
+                4,
+                2
+            ],
+            [
+                6,
+                2
+            ]
+        ],
+        "type": "passive",
+        "spell_id": -1,
+        "img_url": "https://www.wowhead.com/icon=236152/ability-druid-eclipseorange",
```

Comment:
File structure looks good! This is great.

`img_url` need to be urls to the imgs. In this case this is https://wow.zamimg.com/images/wow/icons/large/ability_druid_eclipseorange.jpg